File: platforms/java-desktop/expected/franca/java/ws/WsRouter.java
package franca.java.ws;

import franca.java.expected.Router;
import franca.java.graphics.GraphicsRouter;
import franca.java.skia.SkiaRouter;

public class WsRouter extends Router {

  public WsServer wsServer;
  public GraphicsRouter graphicsRouter;
  public SkiaRouter skiaRouter;

  public WsRouter(GraphicsRouter graphicsRouter, SkiaRouter skiaRouter) {
    super();
    this.graphicsRouter = graphicsRouter;
    this.skiaRouter = skiaRouter;
  }
}

File: platforms/java-desktop/expected/franca/java/ws/WsRouter.kt
package franca.java.ws

import franca.java.expected.Router
import franca.java.graphics.GraphicsRouter
import franca.java.skia.SkiaRouter

class WsRouter(var graphicsRouter: GraphicsRouter, var skiaRouter: SkiaRouter) : Router() {
  var wsServer: WsServer? = null
}
